src/components/MasterItems.js

```diff
@@ -51,6 +65,14 @@ class MasterItems extends Component {
     this.setState({ entries: updatedEntries });
   }
 
+  setQuantity(entryIndex, e) {
+    const updatedEntry = this.state.entries[entryIndex];
+    updatedEntry.quantity = e.target.value;
```

Comment:
Nit / suggestion, but we could combine these two lines as something like this to avoid mutating directly:

`const updatedEntry = R.assoc('quantity', e.target.value, this.state.entries[entryIndex]);`